### PR TITLE
Specify versions for dependencies

### LINKED ISSUE #3: Lock requirement versions
The packages in `requirements.txt` should be set to specific versions to prevent future breaking changes.

Eg:
```
onnxruntime==0.1.4
numpy==1.16.2
Pillow==6.0.0
...
```

### PR DESCRIPTION
**Pull request by MLnick**
Closes #3

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 onnxruntime==0.1.4
-numpy
-Pillow
-mtcnn
-opencv-python
+numpy==1.16.2
+Pillow==5.4.1
```
Comment (bdwyer2) [inline requirements.txt]: Should we use `6.0.0`? Either way is fine with me.

File: requirements.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+mtcnn==0.0.8
+opencv-python==4.0.0.21
```
Comment (bdwyer2) [inline requirements.txt]: Same for this, should we use `4.0.1.24`?

**Comment by MLnick** [comment]
I had just taken whatever versions were from the latest docker container I had built. I can update these versions and see if build is ok.

File: requirements.txt [hunk 3]
```diff
@@ -1,6 +1,6 @@
 onnxruntime==0.1.4
-numpy
-Pillow
-mtcnn
-opencv-python
+numpy==1.16.2
+Pillow==6.0.0
+mtcnn==0.0.8
+opencv-python==4.0.0.24
```
Comment (bdwyer2) [inline requirements.txt]:
```suggestion
opencv-python==4.0.1.24
```

Comment (MLnick) [reply requirements.txt]: woops

**Pull request merged**